index.html

```diff
@@ -183,6 +182,17 @@ <h3>Skills</h3><img src="assets/Union.png" alt="arrowdown">
             </li>
         </ul>
     </section>
+    <section id="contact-me">
+        <h1>Contact me</h1>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that
+            needs coding. I’d love to help with it</p>
+        <form action="https://formspree.io/f/xknkbwpa" method="POST">
+            <input type="text" name="name" placeholder="Name" required>
+            <input type="email" name="_replyto" placeholder="Email" required>
+            <textarea name="message" placeholder="Write your message here" required></textarea>
```

Comment:
Your form looks great in functionality and UX. But there are some form validation elements you have missed. :point_down: 
- [ ] The name field should not accept char length greater than 30.
- [ ] The message field should not accept char length greater than 500.
- [ ] You can use `maxlength` HTML attribute to limit the number of char in a text field. :handshake: